paimon-web-engine/paimon-web-engine-flink/paimon-web-engine-flink-common/src/main/java/org/apache/paimon/web/engine/flink/common/result/SubmissionResult.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.web.engine.flink.common.result;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/** Represents the outcome of a job submission process. */
+public class SubmissionResult implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private final String submitId;
+    private final String jobId;
+    private final String status;
+    private final List<Map<String, Object>> data;
+    private final boolean shouldFetchResult;
```

Comment:
Why does `shouldFetchResult` define in `SubmissionResult`? Did you ever use this in job submission?

Reply:
If the resultType in the result returned by sql gataway is PAYLOAD, it means that the result data can be pulled. If it is EOF, there is no result data and there is no need to pull the data. Use the shouldFetchResult field to indicate whether the data needs to be pulled.